corehq/apps/data_cleaning/templates/data_cleaning/columns/selection.html

```diff
@@ -1,17 +1,26 @@
 {% load django_tables2 %}
 
-<input
-  id="{{ css_id }}"
-  class="form-check-input htmx-disable-on-select-page"
-  type="checkbox"
-  {% if is_checked %}
-    checked="checked"
-  {% endif %}
-  name="selected_record"
-  value="{{ value }}"
+<form
   hx-post="{{ request.path_info }}{% querystring %}"
   hq-hx-action="{{ hq_hx_action }}"
+  hx-trigger="change from:#{{ css_id }}"
   hx-swap="none"
-  hx-disabled-elt="this"
-  {{ attrs.as_html }}
-/>
+  hx-disable-elt="#{{ css_id }}"
```

Comment:
Good - on the last PR I was wondering how rage clicking, or even double clicking, would work.